ui/app/services/namespace.js

```diff
@@ -3,48 +3,54 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { alias, equal } from '@ember/object/computed';
 import Service, { inject as service } from '@ember/service';
```

Comment:
As mentioned in PR #25367, I'm not entirely sure how to amend this when using both services in a Service and extending a service. Maybe something like, `import Service, { service } from '@ember/service'` 🤔

Reply:
That looks right to me! Good catch, I'll ammend